components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsServletResponse.java

```diff
@@ -83,4 +84,8 @@ private HttpServletResponse getResponse() {
         TransientObjectWrapper<HttpServletResponse> transientObjectWrapper = getWrapped();
         return transientObjectWrapper.getWrapped();
     }
+
+    public void addCookie(Cookie cookie){
```

Comment:
@senthalan  there is a formatting issue. Could you fix that?

Reply:
Fixed